presto-main/src/main/resources/webapp/src/components/QueryList.jsx

```diff
@@ -65,6 +66,15 @@ export class QueryListItem extends React.Component {
         return truncateString(formattedQueryText, 300);
     }
 
+    renderWarning() {
+        const query = this.props.query;
+        if (query.warnings && query.warnings.length) {
+            return (
+                <span className="glyphicon glyphicon-stop" style={GLYPHICON_WARNING} data-toggle="tooltip" title="Warning"/>
```

Comment:
Can the tooltip tell us the actual warning codes?

Reply:
I thought about it. But there can be multiple warnings for a single query and I thought it would be too much too show all of them in the tooltip. I can change the code if we want that behavior.